labs/lab0/chapter0.md

```diff
@@ -51,141 +49,80 @@ $ cd ~/
+Now exit out of the minishift VM by disconnecting from the SSH session:
 
 ```bash
 $ exit
 ```
 
+We have two more commands worth mentioning. First off, let's mention `minishift stop`. Stop does exactly what it sounds like and shuts down minishift. However, it does not destroy anything inside just "turns the machine off." If you do want to remove the VM, you can use `minishift delete`. You can spin it right back up again, fresh, with `minishift start`. Finally, we will use `minishift docker-env` in a few minutes to connect the host to the docker daemon in the VM.
+
 ## Container Development Kit (CDK) Walkthrough
 
-The cool thing is, we actually launched the CDK VM by executing `vagrant up` on
-the Vagrantfile in the previous section. The CDK gives a couple options (through the use of 
-[different Vagrantfiles](https://developers.redhat.com/download-manager/file/cdk-2.0.0.zip)) 
-but in this case, we launched the "OpenShift in a Box" VM (aka rhel-ose in 
-the cdk.zip). The CDK provides a simple to use and launch instance of the 
+When we started minishift, we launched the software tooling component of the CDK. As we said before, the CDK provides a lot of support for containerizing your applications. However, the major software tool is minishift.
+
+However, what is minishift? Essentially, it is a simple to use and launch instance of the 
 same OpenShift PaaS you would use at work. Why is a PaaS included in a tool, 
 much less a lab, focused on Containers? Well, the latest version of OpenShift, 
 actually runs Docker Containers to host your "Platform" (in the PaaS sense) 
 and your application.
 
 If you would like to explore the OpenShift Console, you can see it running 
-in your OpenShift instance, if you open a browser. Let's go ahead and try it. 
-
-Open Firefox from the Applications menu and navigate to 
-`https://10.1.2.2:8443/console/`. Once it loads (and you bypass the bad 
-certificate error), you can log in to the console using the default 
-`admin/admin` or to see the less privileged experience, use 
-`openshift-dev/devel` for the `username/password`.
-
-## Setting Up For the Remaining Labs
-
-Let's wrap up the walkthroughs by getting rid of the VM we just
-started. We can do this with a `vagrant destroy` command, which will
-not only shut down a VM, but also remove it completely. Any contents
-of the VM disk images will be lost:
+in your OpenShift instance, if you open a browser. However, before we do that, we need the IP address of the VM minishift created. Easy enough, just run
 
 ```bash
-$ cd ~/summit-2017-container-lab/vagrantcdk
-$ vagrant destroy
-==> default: Removing domain...
+$ minishift ip
+192.168.???.??? 
 ```
+Ok, now we can check out the OpenShift console. Open Firefox from the Applications menu and navigate to `https://<ip>:8443/console/`(replace "<ip>" with the address from the last command). Once it loads (and you bypass the bad certificate error), you can log in to the console using the default `system/admin`.
```

Comment:
in the remaining labs we have the users use developer/developer - might be worth showing them they "can" log in as admin, but then have them log out and log back in a developer.